x-pack/plugins/oss_telemetry/server/lib/collectors/visualizations/get_usage_collector.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { get } from 'lodash';
+import { HapiServer } from '../../../../';
+import { PLUGIN_ID, VIS_TELEMETRY_TASK, VIS_USAGE_TYPE } from '../../../../constants';
+
+export function getUsageCollector(server: HapiServer) {
+  const { taskManager } = server;
+  return {
+    type: VIS_USAGE_TYPE,
+    fetch: async () => {
+      let docs;
+      try {
+        ({ docs } = await taskManager.fetch({
+          query: { bool: { filter: { term: { _id: `${PLUGIN_ID}-${VIS_TELEMETRY_TASK}` } } } },
+        }));
+      } catch (err) {
+        const errMessage = err && err.message ? err.message : err.toString();
+        /*
+         * The usage service WILL to try to fetch from this collector before the task manager has been initialized, because the task manager
+         * has to wait for all plugins to initialize first.
+         * It's fine to ignore it as next time around it will be initialized (or it will throw a different type of error)
+         */
+        if (errMessage.indexOf('NotInitialized') >= 0) {
```

Comment:
`errMessage.includes('NotInitialized')` might read a bit better here (might also be sub-ms faster as well?)

Reply:
Also might want a test for this since it's interesting behavior (not sure how difficult that is)